.github/workflows/test-and-deploy.yml

```diff
@@ -37,19 +37,19 @@ jobs:
       - name: Build & Test
         run: make test
 
-      - name: Install SonarCloud scanner and run analysis
-        if: (github.event_name == 'pull_request' || github.ref_type == 'branch') && !github.event.pull_request.head.repo.fork
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-          TWILIO_ACCOUNT_SID: ${{ secrets.TWILIO_ACCOUNT_SID }}
-          TWILIO_API_KEY: ${{ secrets.TWILIO_CLUSTER_TEST_API_KEY }}
-          TWILIO_API_SECRET: ${{ secrets.TWILIO_CLUSTER_TEST_API_KEY_SECRET }}
-          TWILIO_FROM_NUMBER: ${{ secrets.TWILIO_FROM_NUMBER }}
-          TWILIO_TO_NUMBER: ${{ secrets.TWILIO_TO_NUMBER }}
-        run: |
-          dotnet tool install --global dotnet-sonarscanner
-          make cover
+#      - name: Install SonarCloud scanner and run analysis
```

Comment:
Does [this](https://github.com/twilio/twilio-csharp/pull/717/files) fix not work?

Reply:
Actually the TWILIO_ACCOUNT_SID has expired. That's why we hare disabling it